truetype/truetype.go

```diff
@@ -578,14 +578,16 @@ func parse(ttf []byte, offset int) (font *Font, err error) {
 		return
 	}
 	n, offset := int(u16(ttf, offset)), offset+2
-	if len(ttf) < 16*n+12 {
+	offset = offset + 2*3 // skip 3 short fields
```

Comment:
Change this line to:
offset += 6 // Skip the searchRange, entrySelector and rangeShift.